File: app/src/main/java/com/kaciula/archiman/data/remote/StackExchangeApi.java
package com.kaciula.archiman.data.remote;

import com.kaciula.archiman.data.remote.response.UsersRemote;
import io.reactivex.Observable;
import retrofit2.http.GET;

public interface StackExchangeApi {

  @GET("users?site=stackoverflow")
  Observable<UsersRemote> getUsers();
}

File: app/src/main/kotlin/com/kaciula/archiman/data/remote/StackExchangeApi.kt
package com.kaciula.archiman.data.remote

import com.kaciula.archiman.data.remote.response.UsersRemote
import io.reactivex.Observable
import retrofit2.http.GET

interface StackExchangeApi {

    @GET("users?site=stackoverflow")
    fun getUsers(): Observable<UsersRemote>
}
